src/components/Modals/Send/hooks/useFormSend/useFormSend.tsx

```diff
@@ -61,48 +61,26 @@ export const useFormSend = () => {
         const asset = selectAssetById(store.getState(), sendInput.assetId)!
         console.error(e)
 
-        if ((e as Error).name === 'ResponseError') {
-          const error = await (e as ResponseError).response.json()
-          const jsonError = JSON.parse(error.message)
-
-          const chainAdapterManager = getChainAdapterManager()
-
-          const feeAssetId = chainAdapterManager
-            .get(fromAssetId(asset.assetId).chainId)
-            ?.getFeeAssetId()
-
-          const feeAsset = selectFeeAssetById(store.getState(), feeAssetId ?? '')
-
-          // @TODO: as this is the first time we are handling errors from unchained in this scope, we dont have a complete error handling strategy
-          // If we need to handle more errors, we should create a proper error handler
-          if (jsonError.code === -32000) {
-            toast({
-              title: translate('modals.send.errorTitle', {
-                asset: asset.name,
-              }),
-              description: translate('modals.send.errors.notEnoughNativeToken', {
-                asset: feeAsset?.symbol,
-              }),
-              status: 'error',
-              duration: 9000,
-              isClosable: true,
-              position: 'top-right',
-            })
-          }
-
-          return
-        }
+        const translation =
+          e instanceof ChainAdapterError
+            ? translate(e.metadata.translation, e.metadata.options)
+            : translate('modals.send.errors.transactionRejected')
 
         toast({
           title: translate('modals.send.errorTitle', {
             asset: asset.name,
           }),
-          description: translate('modals.send.errors.transactionRejected'),
+          description: (
+            <InlineCopyButton value={translation}>
+              <RawText>{translation}</RawText>
+            </InlineCopyButton>
```

Comment:
blocking: It doesn't really work as it copy the full error content description

I think we might be able to detect if the description has an address, then put it in the copy button?

Reply:
I went with copying the whole error so it can be easily provided for ops if there are errors. Specifically, the failed the broadcast errors with the corresponding error message from the node.